prisma/schema.prisma

```diff
@@ -0,0 +1,47 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Book {
+  id Int @id @default(autoincrement())
+  title String
+  author Author @relation (fields:[authorId], references:[id])
+  authorId Int @unique
+  isbn Int
+  isRented Boolean
```

Comment:
I'm not sure isRented is necessary. We can infer whether it is or isn't by renterId: you can set renterId to null when it's not rented. As a matter of fact, renterId should be of type `Int?` because a book doesn't *have* to be rented.